dev/io.openliberty.org.apache.myfaces.4.1_fat/fat/src/io/openliberty/org/apache/myfaces41/fat/test/RenderIdTest.java

```diff
@@ -79,22 +88,124 @@ public void setupPerTest() throws Exception {
 
 
     /**
-     *  
-     * Skipped as this is not yet working in 4.1.0-RC1
+     *  Not HTML5, so IDs should NOT be rendered
      *
      * @throws Exception
      */
-    // @Test
-    public void verifyIDsAreGeneratedOnHead() throws Exception {
+    @Test
+    public void verifyIDsAreGenerated_WithIDs_HTML4() throws Exception {
+        try (WebClient webClient = new WebClient()) {
+            URL url = JSFUtils.createHttpUrl(server, APP_NAME, "HTML4_withIDs.xhtml_xhtml");
+
+            HtmlPage page = (HtmlPage) webClient.getPage(url);
+
+            logPage(page);
+
+            assertEquals("Id attribute not found on head!", "", page.getHead().getId());
```

Comment:
Shouldn't the message be "Id attribute was found on head and should not have been!"?